src/components/ec-input-field/ec-input-field.spec.js

```diff
@@ -53,6 +54,11 @@ describe('EcInputField', () => {
     expect(wrapper.element).toMatchSnapshot();
   });
 
+  it('renders properly without note', () => {
+    const wrapper = mountInputField({ note: '' });
```

Comment:
is this the best approach, rather than not passing the prop?

Reply:
thoughts?

Reply:
If we don't pass the prop it will be filled, we want to put it with an empty string because we declared with a string on the function.

I think it can be done in both ways, don't fill on the function and fill in every test or fill on the function and put an empty string on the tests, but I just follow the current implementation.